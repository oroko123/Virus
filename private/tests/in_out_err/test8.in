fajny znaczek 2199 1920 Polska
niefajny znaczek 90210 1921 Argentyna
super znaczek 3000 200 1918 Boliwia
intrygujacy zna321czek 1717 1771 20 3200.84 1915 Nikaragua
dwuznaczek 69 50 1920 Francja
duzy znaczek 08.23 70 1921 RPA
maly znaczek 4-8-15-16-23-42 13,88 1918 Chiny
przecietny znaczek 200 1918 Rosja
jakis-tam-znaczek 340 1915 Finlandia
smutny znaczek :( 1999,99 1918 Grecja
1910 1930