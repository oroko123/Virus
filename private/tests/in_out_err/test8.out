1915 Finlandia 340 jakis-tam-znaczek
1915 Nikaragua 3200.84 intrygujacy zna321czek 1717 1771 20
1918 Boliwia 200 super znaczek 3000
1918 Chiny 13,88 maly znaczek 4-8-15-16-23-42
1918 Grecja 1999,99 smutny znaczek :(
1918 Rosja 200 przecietny znaczek
1920 Francja 50 dwuznaczek 69
1920 Polska 2199 fajny znaczek
1921 Argentyna 90210 niefajny znaczek
1921 RPA 70 duzy znaczek 08.23
